packages/block-library/src/paragraph/edit.native.js

```diff
@@ -53,7 +57,16 @@ function ParagraphBlock( {
 						content: nextContent,
 					} );
 				} }
-				onSplit={ ( value ) => {
+				onSplit={ ( value, keepId ) => {
+					if ( keepId ) {
+						return {
+							...block,
+							attributes: {
+								...block.attributes,
+								content: value || '',
+							},
+						};
+					}
```

Comment:
Same question as above

Reply:
Should we do the same for the web version of this file?

Reply:
Yes, we still need the `keepId`, unless you have a different concept for it.

I already added it to the web version :)